charts/athens-proxy/values.yaml

```diff
@@ -2,7 +2,7 @@ replicaCount: 1
 image:
   registry: docker.io
   repository: gomods/athens
-  tag: v0.11.0
+  # tag: v0.11.0
```

Comment:
Is this supposed to be commented out? It would be better to remove it if it should not be there.

Reply:
Added a comment for clarity. This is for users to override the appVersion of the chart